web/login.php

```diff
@@ -36,6 +36,11 @@ function set_location_header_on_login_success_url(){
 
 /* First check if a custom authentication header was set. */
 $customHeader = ConfigurationParametersManager::getParameter('EXTERNAL_AUTHENTICATION_USER_HEADER');
+
+if (LoginManager::isLogged()) {
```

Comment:
I have a small nitpick that is mostly stylistic. Since we already have a `if/else if` block of checks regarding the login status, perhaps your check would make more sense as an `else if` block at the end of the current block.

Reply:
Done